demo/index.css

```diff
@@ -20,13 +21,18 @@ body {
 .input-row {
     margin-bottom: 20px;
     display: flex;
+    flex-wrap: wrap;
     align-items: center;
 }
 
 .input-row ~ .input-row {
     margin-bottom: 55px;
 }
 
+.input-item {
+    margin-top: 10px;
+}
+
 .input-item ~ .input-item {
     margin-left: 10px;
```

Comment:
Shouldn't we be using `margin: 0 5px` in all inputs so that they are aligned in lower resolutions?

ATM, the first input on the second line has a weird margin-left.